.github/workflows/tests.yaml

```
@@ -0,0 +1,23 @@
+name: Run tests
+
+on: [push, pull_request]
```

can we choose 1 trigger without repeating the same test twice?

```suggestion
on: pull_request
```